backend/server.js

```diff
@@ -44,12 +44,34 @@ app.use(userRoutes);
 
 //GETS
 app.get("/players", async (req, res) => {
-  const { team, orderParam, ascending } = req.query;
+  const { sortType } = req.query;
+  const [feature, ascending] = sortType.split(" ");
+
+  const players = await prisma.player.findMany({
+    orderBy: [
+      {
+        [feature]: ascending,
+      },
+    ],
+  });
 
-  const players = await prisma.player.findMany({});
   res.json(players);
 });
 
+app.get("/searchPlayers", async (req, res) => {
+  const { playerName } = req.query;
+
+  const players = await prisma.player.findMany({
+    where: {
+      player_name: {
+        contains: playerName,
+        mode: "insensitive",
+      },
+    },
+  });
+
+  res.json(players);
+});
```

Comment:
Instead of introducing a new query, does it make sense to instead structure the `/players` query so that we only need one query that takes in both the sort type and a search?